cloud/ovirt/ovirt_disks.py

```diff
@@ -67,6 +67,10 @@
         description:
             - "Format of the disk. Either copy-on-write or raw."
         choices: ['raw', 'cow']
+    sparse:
+        description:
+            - "I(True) create disk with sparse allocation. By default ovirt-engine create sparse allocated disk"
```

Comment:
Can you please change to following description:

- If (True) disk will by createad as sparse, so space will be allocated for the volume as needed, also known as I(thin provision).
- If (False) storage will be allocated right away, also known as I(preallocated).
- Note that this option isn't idempotent as it's not currently possible to change allocation policy of disk via API.